apps/frontend/assets/src/omg-page-api/AdminKeySection.js

```diff
@@ -178,8 +184,9 @@ class ApiKeyPage extends Component {
       privateKeyModalOpen: true
     })
   }
-  onClickAccessKeySwitch = ({ id, expired, fetch }) => async e => {
-    await this.props.updateAccessKey({ id, expired })
+  onClickAccessKeySwitch = ({ id, enabled, fetch }) => async e => {
+    e.stopPropagation()
+    await this.props.enableAccessKey({ id, enabled })
     fetch()
```

Comment:
I think we don't need to fetch here? Can you double check for us ( I might did a mistake or there is a reason for it )

Reply:
it makes the table get into load state :/